docs/tools/CLI/build_profiles.md

```diff
@@ -1,42 +1,42 @@
 <h2 id="build-profiles">Build profiles</h2>
 
-Arm Mbed OS 5 supports three primary build profiles: *develop*, *debug* and *release*. The Online Compiler uses the *develop* profile. When building from Arm Mbed CLI, you can select a profile by adding the `--profile <profile>` flag. You can specify custom user-defined profiles by giving the path to the profile.
+Arm Mbed OS 5 defines three build configurations, each of which is known as a `build profile`. These are collections of toolchain flags used during the build. The three build profiles are *develop*, *debug* and *release*. The Mbed Online Compiler uses the *develop* build profile. When  building from Arm Mbed CLI, you can select the build configuration by adding the `--profile <`**build profile** `name>` flag. You can also specify custom- or user-defined configurations by giving the path to the JSON file defining the build configuration.
```

Comment:
Why use the term "build configuration" only once to define "build profile". This seems like it would only add to the confusion.